### PR TITLE
Update development instructions

### PR DESCRIPTION
**Pull request by findepi**

### CODE REVIEW

**Review by hashhar — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -126,9 +126,10 @@ version with `./setup.py --version`.
 We recommend that you use `virtualenv` for development:
 
 ```
-$ virtualenv /path/to/env
-$ /path/to/env/bin/active
-$ pip install -r requirements.txt
+$ virtualenv .venv
+$ . .venv/bin/activate
+# TODO add requirements.txt: pip install -r requirements.txt
+$ pip install .
```
Comment (losipiuk) [inline README.md:101]: It does not work for me.
What I did is
```
./setup.py install
```

File: README.md [hunk 2]
```diff
@@ -150,6 +150,24 @@ When the code is ready, submit a Pull Request.
 - Prefer code that is readable over one that is "clever".
 - When writing a Git commit message, follow these [guidelines](https://chris.beams.io/posts/git-commit/).
 
+## Releasing
+
+- [Set up your development environment](#Getting-Started-With-Development).
+- Change version in `presto/__init__.py`.
+- Commit and create an annotated tag (`git tag -m '' current_version`)
+- Run the following:
+  ```bash
```
Comment (losipiuk) [inline README.md:159]: nit: a script would be nice to have. Though I will not push on that.

File: README.md [hunk 3]
```diff
@@ -150,6 +150,24 @@ When the code is ready, submit a Pull Request.
...
+  . .venv/bin/activate &&
+  pip install twine &&
+  rm -rf dist/ &&
+  ./setup.py sdist bdist_wheel &&
+  twine upload dist/* &&
+  open https://pypi.org/project/presto-client/ &&
+  echo "Released!"
+  ```
+- Push the branch and the tag (`git push upstream master current_version`)
+- Send release announcement.
```
Comment (losipiuk) [inline README.md:169]: link to example release annoucment?

Comment (losipiuk) [reply README.md:169]: Do we also add new release to github releases page.?

**Review by losipiuk — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (findepi) [reply README.md:169]: follow-up improvement

File: README.md (hunk 2 above)
Comment (findepi) [reply README.md:159]: follow-up improvement

File: README.md (hunk 1 above)
Comment (findepi) [reply README.md:101]: `pip --version` ?

Comment (findepi) [reply README.md:101]: > It does not work for me.

in what way it does not?

Comment (losipiuk) [reply README.md:101]: Oh - it worked now :)

**Pull request merged**